e2e/handles/handles.test.ts

```diff
@@ -61,20 +59,18 @@ describe("🤝 Handles", () => {
             await ExtrinsicHelper.runToBlock(currentBlock + expirationOffset + 1); // Must be at least 1 > original expiration
 
             const retireHandle = ExtrinsicHelper.retireHandle(msaOwnerKeys);
-            const [event] = await retireHandle.fundAndSend(fundingSource);
+            const { target: event } = await retireHandle.fundAndSend(fundingSource);
             assert.notEqual(event, undefined, "retireHandle should return an event");
-            if (event && retireHandle.api.events.handles.HandleRetired.is(event)) {
-                let handle = event.data.handle.toString();
-                assert.notEqual(handle, "", "retireHandle should return the correct handle");
-            }
+            const handle = event!.data.handle.toString();
+            assert.notEqual(handle, "", "retireHandle should return the correct handle");
         });
     });
 
     describe("@Alt Path: Claim Handle with possible presumptive suffix/RPC test", () => {
         /// Check chain to getNextSuffixesForHandle
 
         it("should be able to claim a handle and check suffix (=suffix_assumed if available on chain)", async function () {
-            const handle = "test1";
+            const handle = "test-" + Math.random().toFixed(10).toString().replaceAll("0.", "");
```

Comment:
While not completely foolproof, this enables our tests to run more than 90 times on Testnet